larq_zoo/literature/binary_alex_net.py

```diff
@@ -128,6 +128,12 @@ def BinaryAlexNet(
     /plots/binary_alexnet.vg.json
     ```
 
+    # ImageNet Metrics
+    Top-1 Accuracy: 36.30 %
+    Top-5 Accuracy: 61.53 %
+    Parameters: 61 859 192
+    Memory: 7.49 MB
+
```

Comment:
What do you think about rendering it as a table, so it will show up nicely on the website similar to https://docs.larq.dev/zoo/#available-models?
```suggestion
    | Top-1 Accuracy | Top-5 Accuracy | Parameters | Memory  |
    | -------------- | -------------- | ---------- | ------- |
    | 36.30 %        | 61.53 %        | 61 859 192 | 7.49 MB |
```

Reply:
Agreed. Pushed the changes